test/IXSocketTest.cpp

```diff
@@ -95,4 +95,19 @@ TEST_CASE("socket", "[socket]")
         testSocket(host, port, request, socket, expectedStatus, timeoutSecs);
     }
 #endif
+
+TEST_CASE("isValidIpAddress") {
```

Comment:
I don't know if this works, I am not able to compile the test folder under windows.  I also tried under Centos7 with build_linux.sh, but some files are missing.  So, I don't know if I am supposed to build....